tests/test_logging.py

```diff
@@ -20,6 +22,44 @@ def assert_records(records):
     return details
 
 
+def test_initialization_from_ini(caplog, tmpdir):
+    ini_content = textwrap.dedent("""
+    [loggers]
+    keys = root, test_logger
+
+    [handlers]
+    keys = console
+
+    [formatters]
+    keys=json
+
+    [logger_root]
+    level = INFO
+    handlers = console
+
+    [logger_test_logger]
+    level = DEBUG
+    handlers = console
+    qualname = test_logger
+
+    [handler_console]
+    class=StreamHandler
+    level=DEBUG
+    args = (sys.stderr,)
+    formatter=json
+
+    [formatter_json]
+    class = dockerflow.logging.JsonLogFormatter
+    """)
+    ini_file = tmpdir.join('logging.ini')
+    ini_file.write(ini_content)
+    logging.config.fileConfig(str(ini_file))
+    logging.info('I am logging in mozlog format now! woo hoo!')
+    logger = logging.getLogger('test_logger')
```

Comment:
Wouldn't it work with the `root` logger? In other words, what do you think about reducing the ini to the bare minimum?

Reply:
Yeah, I tried it with a root logger as well, lemme configure that.